lua/telescope/builtin.lua

```diff
@@ -45,14 +46,18 @@ builtin.git_files = function(opts)
     opts.cwd = vim.fn.expand(opts.cwd)
   end
 
+  opts.track_results = vim.g["telescope_cache_results"]
+  local use_prime = vim.g["telescope_prime_fuzzy_find"]
```

Comment:
If you wanted to do this for yourself, you should set a new value in `config.lua`

Probably something like `file_matching_strategy` or something, it can be set to `prime` to use your matcher.